pep_rss_gen.py

```diff
@@ -0,0 +1,137 @@
+import datetime
```

Comment:
If we merge #1679 , we will have a "duplicated" code with your PR. Is it right?

Reply:
Yes and no -- the abstract code for this was taken/lifted from @hugovk's work on #1679, but the long term aim if this series of PRs is merged would be to remove a lot of the old code in the repo - such as `pep2rss.py` and `PyRSS2Gen.py`, as the new system would be hooked into the Sphinx process. 

Happy to do whatever leads to the best improvements though?

Reply:
More specifically L61-78 came from 1679, not sure how to pull in work from another PR via git to assign credit in the history though!


https://github.com/AA-Turner/peps/blob/371d794ff7f5cd97abbdbfcb16bf8572ad9017ce/pep_rss_gen.py#L61-L78

Reply:
(Don't worry about assigning credit to me for anything taken from #1679!)

Reply:
Ahah thanks!